### ISSUE TITLE
Swift support

### ISSUE THREAD

**Issue by NikolaiSa** [ryanolsonk/LLDB-QuickLook#9]
Is this supported for swift? I get the below error but when I call snap.quickLookDebugData() it works, so it clearly is accessible.

![70272cee-5f32-431e-bbf3-0c94095395f6](https://user-images.githubusercontent.com/746234/38339215-ff2d292a-3821-11e8-88e1-b0fcab6f93f4.png)

**Comment by dxclancy** [comment]
Is quickLookDebugData your method? Is it/maybe it has to be declared @objc?

**Comment by NikolaiSa** [comment]
No, it's the method that is declared in `UIView+LLDBQuickLook.m` so it is already objective-c. I only added it to the header as well so I can call it from the command line.

The image `snap` I want to use for quicklook is declared in swift. I added `UIView+LLDBQuickLook.h` to my bridging header
